packages/fxa-settings/e2e/intern.js

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const fs = require('fs');
+const intern = require('intern').default;
+const args = require('yargs').argv;
+const firefoxProfile = require('./tools/firefox_profile');
+
+// Tests
+const testsMain = require('./functional');
+
+const fxaAuthRoot = args.fxaAuthRoot || 'http://localhost:9000/v1';
+const fxaContentRoot = args.fxaContentRoot || 'http://localhost:3030/';
+const fxaOAuthRoot = args.fxaOAuthRoot || 'http://localhost:9000';
+const fxaProfileRoot = args.fxaProfileRoot || 'http://localhost:1111';
+const fxaTokenRoot = args.fxaTokenRoot || 'http://localhost:5000/token';
+const fxaEmailRoot = args.fxaEmailRoot || 'http://localhost:9001';
+const fxaOAuthApp = args.fxaOAuthApp || 'http://localhost:8080/';
+const fxaUntrustedOauthApp =
+  args.fxaUntrustedOauthApp || 'http://localhost:10139/';
+const fxaPaymentsRoot = args.fxaPaymentsRoot || 'http://localhost:3031/';
+const output = args.output || 'test-results.xml';
+
+const fxaToken = args.fxaToken || 'http://';
+const asyncTimeout = parseInt(args.asyncTimeout || 5000, 10);
+
+// On Circle, we bail after the first failure.
+// args.bailAfterFirstFailure comes in as a string.
+const bailAfterFirstFailure = args.bailAfterFirstFailure === 'true';
+
+const testProductId = args.testProductId || 'prod_GqM9ToKK62qjkK';
+
+// Intern specific options are here: https://theintern.io/docs.html#Intern/4/docs/docs%2Fconfiguration.md/properties
+const config = {
+  asyncTimeout: asyncTimeout,
+  bail: bailAfterFirstFailure,
+  defaultTimeout: 45000, // 30 seconds just isn't long enough for some tests.
```

Comment:
I kind of wonder if we should lower this, and when we have to increase it, we'll know which tests are causing the uppage?